watchtower-plugin/src/main.rs

```diff
@@ -517,7 +531,11 @@ async fn main() -> Result<(), Error> {
             "watchtower-proxy",
             Value::OptString,
             "Socks v5 proxy IP address and port for the watchtower client",
-        ))
+        )).option(ConfigOption::new(
+            "watchtower-auto-retry-delay",
+            Value::Integer(3600),
+            "the time (in seconds) that a retrier will wait before auto-retrying a failed tower",
+         ))
```

Comment:
Should the default for this option be really an hour?
I think a half a day to 10 days auto retry period would be a sane default.

Reply:
Yeah, this doesn't need to be this low. I will default it to once a day and may increase/decrease it if we get any meaningful feedback from users